MediaWikiTelegramBot.js

```diff
@@ -1,20 +1,26 @@
+// Require Telegram API & http request
 var TelegramBot = require( 'node-telegram-bot-api' );
 var request = require( 'request' ).defaults( {
     jar: true
 } );
+
+// Require yaml fs reader
```

Comment:
Either:

- no comments at all, because these lines are easy to understand
- 1 or 2 comments that explain why are we loading these packages.